store/sqlstore/upgrade.go

```diff
@@ -353,6 +353,9 @@ func UpgradeDatabaseToVersion47(sqlStore SqlStore) {
 }
 
 func UpgradeDatabaseToVersion48(sqlStore SqlStore) {
+
+	sqlStore.CreateColumnIfNotExists("Teams", "LastTeamIconUpdate", "bigint", "bigint", "0")
```

Comment:
If you wouldn't mine moving that into the commented out if statement, that'd be helpful although whoever uncomments that during upgrade should probably know to move it inside anyway